### PR TITLE
ENG-1457 Add content, page, and widget config related to About Us page

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **alepintus**

If they are not already assigned, you can assign the PR to them by writing `/assign @alepintus` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/resources/component/defaultResources/postprocess/TCL/content1.xml
Comment (gidesan) [inline src/main/resources/component/defaultResources/postprocess/TCL/content1.xml]:
```suggestion
The company,  founded in 2010 as an open-source system integrator, was re-founded as a product company in 2015 in response to the growing demand for tools and services  to create modern online experiences. Since then, Entando has stepped into international markets. It’s headquartered in San Diego, California with R&D and sales offices in Europe, and features teams all over the world, including the United States, Italy, Brazil, South Africa, Georgia and the Philippines.
```

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
<property key="it">Chi Siamo</property>
```

File: src/main/resources/sql/defaultResources/port_data_production.sql
Comment (gidesan) [inline src/main/resources/sql/defaultResources/port_data_production.sql]:
```suggestion
<property key="it">Chi Siamo</property>
```

**Review by gidesan — CHANGES_REQUESTED** [review]

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-de-app/pull/132#pullrequestreview-527218796" title="Approved">gidesan</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **alepintus**

If they are not already assigned, you can assign the PR to them by writing `/assign @alepintus` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
